security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
 
 For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
 
+[id="compliance-operator-release-notes-1-8-0_{context}"]
+== OpenShift Compliance Operator 1.8.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.8.0:
+
+* link:https://access.redhat.com/errata/RHBA-2025:XXXX[RHBA-2025:XXXX - OpenShift Compliance Operator 1.8.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-8-0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* With this update, the Compliance Operator provides the Collective Evidence Language (CEL) scanner in TECH PREVIEW status. The CEL scanner implements a new `CustomRule` Custom Resource Definition (CRD) that allows administrators to define and enforce custom security policies using CEL expressions. This new content format adds to (and does not replace) the existing XCCDF (Extensible Configuration Checklist Description Format) profiles, providing more ability to comply with custom security policies. For more information, see (link:https://issues.redhat.com/browse/CMP-3118[*CMP-3118*]).
+
+* Previously, Compliance Operator required persistent storage to save raw scan results, which presented challenges for edge deployments and environments without storage infrastructure. With this release, Compliance Operator supports running scans without persistent storage. Administrators can set `rawResultStorage.enabled: false` in `ScanSetting` resources to disable storage of scan result files, allowing compliance scans to run in storage-constrained environments such as edge deployments and Single Node OpenShift. Compliance check results remain fully available through `ComplianceCheckResult` resources. Raw result storage remains enabled by default for backward compatibility. For more information, see (link:https://issues.redhat.com/browse/CMP-1225[*CMP-1225*]).
+
+* Previously, Compliance Operator was able to scan using the BSI standards required in the EU. With this release, an RHCOS profile has now been made available which is BSI compliant. For more information, see (link:https://issues.redhat.com/browse/CMP-3720[*CMP-3720*]).
```

Comment:
Maybe better to be more specific:
"Previously,  Compliance Operator provided ocp4-bsi and ocp4-bsi-node profiles for BSI compliance scanning. This release introduces the rhcos4-bsi profile, extending BSI standard coverage to RHCOS systems."